crates/language/src/language_registry.rs

```diff
@@ -167,18 +164,12 @@ impl AvailableLanguage {
     }
 }
 
-#[derive(Copy, Clone, Default, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
These implementations are no longer needed because I removed the `<` enum comparison in favor an exhaustive `match`.